inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/SinkType.java

```diff
@@ -21,14 +21,15 @@
 
 public enum SinkType {
 
-    HIVE, KAFKA, ICEBERG, CLICKHOUSE, HBASE, POSTGRES, ELASTICSEARCH, SQLSERVER;
+    HIVE, KAFKA, ICEBERG, CLICKHOUSE, HBASE, POSTGRES, ELASTICSEARCH, SQLSERVER, HDFS;
 
     public static final String SINK_HIVE = "HIVE";
+    public static final String SINK_HDFS = "HDFS";
```

Comment:
It is recommended to keep the order of the constants as in the enum instance above.

Reply:
doone